mapsforge-map/src/main/java/org/mapsforge/map/layer/renderer/DatabaseRenderer.java

```diff
@@ -163,83 +163,96 @@ private TileBitmap createBackgroundBitmap(RenderContext renderContext) {
             renderContext.canvasRasterer.fill(renderContext.renderTheme.getMapBackgroundOutside());
         }
         return bitmap;
-
     }
 
-    private Set<MapElementContainer> processLabels(RenderContext renderContext) {
-        // if we are drawing the labels per tile, we need to establish which tile-overlapping
-        // elements need to be drawn.
-        Set<MapElementContainer> labelsToDraw = new HashSet<>();
-
-        synchronized (tileDependencies) {
-            // first we need to get the labels from the adjacent tiles if they have already been drawn
-            // as those overlapping items must also be drawn on the current tile. They must be drawn regardless
-            // of priority clashes as a part of them has alread been drawn.
-            Set<Tile> neighbours = renderContext.rendererJob.tile.getNeighbours();
-            Iterator<Tile> tileIterator = neighbours.iterator();
-            Set<MapElementContainer> undrawableElements = new HashSet<>();
-
-            tileDependencies.addTileInProgress(renderContext.rendererJob.tile);
-            while (tileIterator.hasNext()) {
-                Tile neighbour = tileIterator.next();
-
-                if (tileDependencies.isTileInProgress(neighbour) || tileCache.containsKey(renderContext.rendererJob.otherTile(neighbour))) {
-                    // if a tile has already been drawn, the elements drawn that overlap onto the
-                    // current tile should be in the tile dependencies, we add them to the labels that
-                    // need to be drawn onto this tile. For the multi-threaded renderer we also need to take
-                    // those tiles into account that are not yet in the TileCache: this is taken care of by the
-                    // set of tilesInProgress inside the TileDependencies.
-                    labelsToDraw.addAll(tileDependencies.getOverlappingElements(neighbour, renderContext.rendererJob.tile));
-
-                    // but we need to remove the labels for this tile that overlap onto a tile that has been drawn
-                    for (MapElementContainer current : renderContext.labels) {
-                        if (current.intersects(neighbour.getBoundaryAbsolute())) {
-                            undrawableElements.add(current);
+    private void processLabels(RenderContext renderContext) {
+        final List<MapElementContainer> labelsToDraw;
+        {
```

Comment:
We can avoid this `{` so commit history is more clear with the changes.

Reply:
Those are scope-reducing braces, so it's clear that the purpose of the rest of the code is to set the variable. In our opinion it is better to have more error-resistant code than not to have it. No problem to remove those, are you sure?